lib/phoenix/pubsub/adapter.ex

```diff
@@ -0,0 +1,36 @@
+defmodule Phoenix.PubSub.Adapter do
```

Comment:
This is gone right? We have moved to a message protocol so this is not used for anything.